components/accordion-menu/style.css

```
@@ -0,0 +1,44 @@
+gameface-accordion-menu {
+    background-color: white;
+    box-shadow: 0 2px 10px 0 rgba(0, 0, 0, 0.5);
+}
+
+gameface-accordion-panel {
+    border-bottom: 1px solid #dddddd;
+}
+
+.accordion-last-panel {
+    border-bottom: 0;
+}
+
+gameface-accordion-header {
+    padding: 16px 24px;
+    position: relative;
+}
+
+gameface-accordion-content {
+    padding: 0 24px;
+    height: 0;
+    overflow: hidden;
+    transition: height 0.2s;
+}
+
+gameface-accordion-menu, gameface-accordion-panel, gameface-accordion-header, gameface-accordion-content {
+    display: block;
+}
+
+.accordion-panel-disabled {
+    color: #dddddd;
+}
+
+.accordion-header-caret {
+    position: absolute;
+    transform: rotate(0deg);
+    transition-property: transform;
+    transition-duration: 0.4s;
+    border-top: 6px solid #dddddd;
+    border-left: 6px solid transparent;
+    border-right: 6px solid transparent;
+    right: 24px;
+    top: 24px;
```

```suggestion
    top: 24px;
    pointer-events: none;
```

So the event can pass through and the element below could receive the click.